### PR TITLE
feat: app bridge theme dev

### PR DESCRIPTION
**Pull request by anxobotana**

### CODE REVIEW

File: packages/app-bridge-theme/src/types/Context.ts [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    defaultLanguage: string;
+    isEditing: boolean;
+    isPublicLink: boolean;
+    isAuthenticated: boolean;
+    isSearchDialogOpen: boolean;
+    languages: Language[];
+    template: TemplateContext | null;
+    settings: {
+        templateSettings: Record<string, unknown>;
+        templateAssets: Record<string, unknown>;
```
Comment (mike85) [inline packages/app-bridge-theme/src/types/Context.ts:35]: We could make it more specific here: `Record<string, Asset[]>`

File: packages/app-bridge-theme/src/types/Guideline.ts [hunk 2]
```diff
@@ -1,6 +1,15 @@
 /* (c) Copyright Frontify Ltd., all rights reserved. */
 
-import { type LinkSettingsDisplay, type LinkSettingsIconPosition } from './Document';
+enum LinkSettingsDisplay {
```
Comment (mike85) [inline packages/app-bridge-theme/src/types/Guideline.ts:3]: Shouldn’t we export these types to make it possible to import in web-app?

File: packages/app-bridge-theme/vite.config.mts [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    test: {
+        css: true,
+        coverage: {
+            all: true,
+            reporter: ['text', 'lcov'],
+            include: ['src/**/*.ts', 'src/**/*.tsx'],
+            exclude: ['src/**/spec.ts', 'src/**/spec.tsx'],
+        },
+        setupFiles: ['./src/setupTests.ts'],
+    },
```
Comment (oliverschwendener) [inline packages/app-bridge-theme/vite.config.mts:41]: Remove

File: .github/workflows/app-bridge-theme-continuous-integration.yml [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+              run: pnpm i --frozen-lockfile
+
+            - name: Typecheck code
+              run: pnpm --stream --filter {packages/app-bridge-theme} typecheck
+
+            - name: Lint code
+              run: pnpm --stream --filter {packages/app-bridge-theme} lint
+
+            - name: Test code
+              run: pnpm --stream --filter {packages/app-bridge-theme} test:coverage
```
Comment (Kenny806) [inline .github/workflows/app-bridge-theme-continuous-integration.yml:46]: We currently don't have any tests or code to be tested -> Remove this step

File: packages/app-bridge-theme/package.json [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+    },
+    "scripts": {
+        "build": "ts-json-as-const ./package.json && vite build",
+        "dev": "vite build --watch",
+        "lint": "eslint .",
+        "lint:fix": "eslint --fix .",
+        "test": "vitest run",
+        "test:ui": "vitest --ui",
+        "test:watch": "vitest",
+        "test:coverage": "vitest run --coverage",
```
Comment (Kenny806) [inline packages/app-bridge-theme/package.json:37]: We currently don't have any tests or code to be tested -> Remove

File: packages/app-bridge-theme/package.json [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+        "eslint": "^8.57.0",
+        "eslint-plugin-notice": "^0.9.10",
+        "prettier": "^3.2.5",
+        "react": "^18.2.0",
+        "ts-json-as-const": "^1.0.7",
+        "type-fest": "^4.15.0",
+        "typescript": "^5.4.4",
+        "vite": "^5.2.10",
+        "vite-plugin-dts": "^3.8.1",
+        "vitest": "^1.4.0"
```
Comment (Kenny806) [inline packages/app-bridge-theme/package.json:61]: We currently don't have any tests or code to be tested -> Remove test-related dependencies

File: packages/app-bridge-theme/sonar-project.properties [hunk 7]
```diff
@@ -0,0 +1,6 @@
+sonar.projectKey=frontify_brand-sdk_app-bridge-theme
+sonar.organization=frontify
+
+sonar.javascript.lcov.reportPaths=./coverage/lcov.info
+sonar.coverage.exclusions=src/tests/**/*,src/types/**/*,src/**/*.spec.ts,src/**/*.spec.tsx
```
Comment (Kenny806) [inline packages/app-bridge-theme/sonar-project.properties:5]: We currently don't have any tests or code to be tested -> Remove

File: packages/app-bridge-theme/tsconfig.json [hunk 8]
```diff
@@ -0,0 +1,24 @@
...
+        "jsx": "react-jsx",
+        "strict": true,
+        "noUncheckedIndexedAccess": false,
+        "noUnusedLocals": true,
+        "noUnusedParameters": true,
+        "noFallthroughCasesInSwitch": true,
+        "baseUrl": ".",
+        "types": ["vite/client"]
+    },
+    "include": ["src", "./setupTests.ts"],
```
Comment (Kenny806) [inline packages/app-bridge-theme/tsconfig.json:22]: We currently don't have any tests or code to be tested -> Remove setupTests.ts

File: .github/workflows/app-bridge-theme-continuous-integration.yml [hunk 9]
```diff
@@ -0,0 +1,54 @@
...
+
+            - name: SonarCloud Scan
+              uses: sonarsource/sonarcloud-github-action@master
+              env:
+                  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+                  SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
+              with:
+                  projectBaseDir: ./packages/app-bridge-theme
```
Comment (oliverschwendener) [inline .github/workflows/app-bridge-theme-continuous-integration.yml:54]: As we don't have a SonarCloud project we can remove this, right?

File: packages/app-bridge-theme/vite.config.mts [hunk 10]
```diff
@@ -0,0 +1,42 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import dts from 'vite-plugin-dts';
+import { defineConfig } from 'vitest/config';
```
Comment (Kenny806) [inline packages/app-bridge-theme/vite.config.mts:4]: We currently don't have any tests or code to be tested -> Remove

File: packages/app-bridge-theme/.eslintignore [hunk 11]
```diff
@@ -0,0 +1 @@
+
```
Comment (oliverschwendener) [inline packages/app-bridge-theme/.eslintignore:1]: I guess we can remove this empty file

File: packages/app-bridge-theme/package.json [hunk 12]
```diff
@@ -0,0 +1,63 @@
...
+        "lint:fix": "eslint --fix .",
+        "test": "vitest run",
+        "test:ui": "vitest --ui",
+        "test:watch": "vitest",
+        "test:coverage": "vitest run --coverage",
+        "typecheck": "tsc --noEmit"
+    },
+    "peerDependencies": {
+        "react": "^18",
+        "sinon": "^15 || ^16 || ^17"
```
Comment (oliverschwendener) [inline packages/app-bridge-theme/package.json:42]: As we removed "sinon"  as a dependency, I guess we can remove this as well.

```suggestion
```

File: packages/app-bridge-theme/sonar-project.properties [hunk 13]
```diff
@@ -0,0 +1,6 @@
...
+sonar.cpd.exclusions=**/*.spec.ts,**/*.spec.tsx
```
Comment (oliverschwendener) [inline packages/app-bridge-theme/sonar-project.properties:6]: Do we have a SonarCloud project? If not, we can remove this file.

File: packages/app-bridge-theme/package.json [hunk 14]
```diff
@@ -0,0 +1,55 @@
...
+        "lint:fix": "eslint --fix .",
+        "typecheck": "tsc --noEmit"
+    },
+    "peerDependencies": {
+        "react": "^18"
+    },
+    "dependencies": {},
+    "devDependencies": {
+        "@frontify/eslint-config-react": "^0.17.6",
+        "@testing-library/react": "^14.3.0",
```
Comment (Kenny806) [inline packages/app-bridge-theme/package.json:42]: This can also be removed

**Review by oliverschwendener — APPROVED** [review]

**Review by Kenny806 — APPROVED** [review]
LGTM

**Review by mike85 — APPROVED** [review]

**Pull request merged**